R/object_name_linters.R

```diff
@@ -56,7 +56,9 @@ object_name_linter <- function(styles = c("snake_case", "symbols")) {
       declared_s3_generics(xml),
       namespace_imports(find_package(source_file$filename))$fun,
       names(.knownS3Generics),
-      .S3PrimitiveGenerics, ls(baseenv())))
+      .S3PrimitiveGenerics, ls(baseenv())
+    ))
+    generics <- unique(generics[generics != ""])
```

Comment:
I prefer `generics[nzchar(generics)]`, not sure how stylistic that is though

Reply:
I don't mind either way, but I'd still keep the call to `unique()`